src/content/docs/mobile-monitoring/new-relic-mobile-flutter/monitor-your-flutter-application.mdx

```diff
@@ -0,0 +1,431 @@
+
+<img
+  title="mobile UI for flutter "
+  alt="mobile UI for flutter "
+  src={mobileFlutterOverview}
+/>
+
+<figcaption>
+  
+</figcaption>
+
+Ready to get started? Follow the steps below! 
+
+
+## Step 1: Review the requirements
+
+Before you start the install process, make sure your Flutter app meets these version requirements:
+
+* Flutter 2.5 and higher.
+* Dart 2.16.2 and higher, up to 2.9.9
+
+For Android-native apps:
+    * Android API 24+
+    * See [Android native requirements](/docs/mobile-monitoring/new-relic-mobile-android/get-started/new-relic-android-compatibility-requirements)
+
+For iOS-native apps:
+    * See [IOS native requirements](/docs/mobile-monitoring/new-relic-mobile-ios/get-started/new-relic-ios-compatibility-requirements)
+
+## Step 2: Install the Flutter agent 
+
+First, you'll need to install the New Relic plugin into your dart project. In your `pubspec.yaml`, add the following to `dependencies`: 
+
+```dart
+dependencies:
+    newrelic_mobile: 0.0.1-dev.10
+```
+
+## Step 3: Copy your application token #app-token
+
+The application token is used for New Relic to authenticate your Flutter app's data. 
+
+To view and copy your application token in the New Relic UI:
+
+1. Go to **[one.newrelic.com](https://one.newrelic.com/all-capabilities)**, click **Add data**, then click **Mobile**.
+2. Select your Flutter app. 
+3. Go to **Settings > Application** and copy the displayed **Application token**.
+
+You'll add the token to configuration code in the next step. 
+
+## Step 4: Configure your Flutter project
+
+In your Flutter project, open `main.dart` and add the following code:
+
+```dart
+import 'package:newrelic_mobile/newrelic_mobile.dart';
+
+    var appToken = "";
+
+    if (Platform.isAndroid) {
+        appToken = "<android app token>"; // Replace with your application token copied from the New Relic UI. 
+    } else if (Platform.isIOS) {
+        appToken = "<ios app token>"; // Replace with your application token copied from the New Relic UI. 
+    }
+
+    Config config =
+        Config(accessToken: appToken,
+
+        //Android Specific
+        // Optional: Enable or disable collection of event data.
+        analyticsEventEnabled: true,
+
+        // Optional:Enable or disable reporting successful HTTP requests to the MobileRequest event type.
+        networkErrorRequestEnabled: true,
+
+        // Optional:Enable or disable reporting network and HTTP request errors to the MobileRequestError event type.
+        networkRequestEnabled: true,
+
+        // Optional:Enable or disable crash reporting.
+        crashReportingEnabled: true,
+
+        // Optional:Enable or disable interaction tracing. Trace instrumentation still occurs, but no traces are harvested. This will disable default and custom interactions.
+        interactionTracingEnabled: true,
```

Comment:
```suggestion
        // Optional: Enable or disable reporting successful HTTP requests to the MobileRequest event type.
        networkErrorRequestEnabled: true,

        // Optional: Enable or disable reporting network and HTTP request errors to the MobileRequestError event type.
        networkRequestEnabled: true,

        // Optional: Enable or disable crash reporting.
        crashReportingEnabled: true,

        // Optional: Enable or disable interaction tracing. Trace instrumentation still occurs, but no traces are harvested. This will disable default and custom interactions.
        interactionTracingEnabled: true,
```